draft-irtf-cfrg-vrf.xml

```diff
@@ -1041,9 +1041,10 @@
                     <t>let c_string = pi_string[ptLen]...pi_string[ptLen+n-1]</t>
                     <t>let s_string =pi_string[ptLen+n]...pi_string[ptLen+n+qLen-1]</t>
                     <t>Gamma = string_to_point(gamma_string)</t>
-                    <t>if Gamma = "INVALID" output "INVALID" and stop.</t>
```

Comment:
Removed the trailing period on this line for consistency with the rest of the document. This was the only one that ended in a period.